docs/photoniq/vwrs/01-how-vwrs-works.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How VWRs Works
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) are an advanced prioritization service that manages the flow of visitors entering a site, especially during periods of high demand. It serves as a protective layer for your origin server, effectively managing high traffic volumes to prevent server overload.
+
+## Solution Overview
+
+![VWR Solution Architecture](../../../static/img/photoniq/vwr/vwr-solution-architecture.png)
+
+The waiting room solution includes these essential aspects:
+
+- **Origin Service**: The website for which you want to set up a virtual waiting room.
+- **Origin Metrics Service**: Allows you to retrieve information about a specific domain.
+- **Waiting Room Page**: The endpoint (URL) with the waiting room attached.
+
+## The VWR Workflow
+
+During moments of peak traffic, VWRs makes a new page separate from your origin to organize visitors. The VWRs workflow ensures that users are directed to the appropriate origin service while ensuring fairness in the queue and providing metrics and alerts to the administrator.
+
+1. The Akamai EdgeWorkers check the virtual room service to confirm if a waiting room is set up for an origin service
+
+    - If no waiting room, the EdgeWorker sends the traffic request directly to the origin service.
```

Comment:
Here it is "waiting room" and above it is "virtual room service"

Reply:
Should that not be:

"if no waiting room is configured". or is it ok to just say "if no waiting room"?